elements/timeline-view/scrolling-box.html

```diff
@@ -72,16 +76,27 @@
                     duration: 0,
                     easing: "linear",
                 });
+
+            const drawer = document.getElementsByTagName("paper-drawer-panel");
+            let containerEl = drawer[0];
```

Comment:
El containeros needs a longer name, this isn't the first time we've run into this.